homework/src/index.js

```diff
@@ -16,6 +15,98 @@
     xhr.send();
   }
 
+  function repoData(error, data) {
+    if (error !== null) {
+      console.error(error);
```

Comment:
You should not print the error to the console as the user would never know what went wrong. Instead, you should show the error on the webpage